build/arm-tests/3d-0m-3c-ext-kp-location.json

```diff
@@ -4,7 +4,7 @@
 	"why" : "",
 	"location" : "westeurope",
 	"parameters" : {
-		"loadBalancerType":{"value":"external"},
+		"loadBalancerType":{"value":"External"},
```

Comment:
I was thinking along the lines of consistency; we have `Yes/No` options for kibana and loadbalancer, but then `Internal/Eternal` for load balancer type and `new/existing` for new or existing virtual network. 

Can we lowercase all of the options?